helm/gorush/templates/redis-deployment.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis
+  namespace: {{ .Chart.Name }}
+  labels:
+    {{- include "gorush.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: redis
+      role: master
+      tier: backend
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: redis
+        role: master
+        tier: backend
+    spec:
+      containers:
+        - name: master
+          image: redis
+          ports:
+            - containerPort: {{ .Values.redis.port }}
```

Comment:
Please use an existing [Redis chart](https://github.com/bitnami/charts/tree/master/bitnami/redis) and include it as an optional dependency. 

As gorush doesn't mandate that we use Redis (we can use many backends) this needs to be optional for the user to enable.

```suggestion
```

Reply:
fixed